src/main/kotlin/calculator/domain/StringAddCalculator.kt

```diff
@@ -0,0 +1,14 @@
+package calculator.domain
+
+import calculator.parser.StringSplitModule
+
+class StringAddCalculator(
+    private val stringSplitModule: StringSplitModule = StringSplitModule()
+) {
+
+    fun add(text: String?): Int {
+        if (text.isNullOrEmpty() || text.isNullOrBlank()) return 0
+
+        return stringSplitModule.split(text).sumOf { it }
```

Comment:
```suggestion
        return stringSplitModule
            .split(text)
            .sumOf { it }
```
함수 체이닝 컨벤션을 적용해주시면 좋을 것 같아요

ref. https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls